Kinvey/Kinvey/FindOperation.swift

```diff
@@ -294,8 +179,32 @@ internal class FindOperation<T: Persistable>: ReadOperation<T, AnyRandomAccessCo
                                     reject(error)
                                 }
                             }
+                        } else {
+                            reject(Error.invalidResponse(httpResponse: nil, data: nil))
                         }
-                    } else {
+                    case .failure(let error):
```

Comment:
In case of failure, we need to decide if the SDK should initiate a regular `fetch` automatically, or whether we should throw an error to the developer and leave it up to them to do it.

Reply:
i don't think we should do a regular fetch in case of failure, this can lead users to assume the delta set is working when is actually not. If delta set is enable i think we should use delta set and inform the user in case of failure